test/integration/security-rules.spec.ts

```diff
@@ -105,23 +103,40 @@ describe('admin.securityRules', () => {
     });
   });
 
+  function getRandomInt(max: number) {
+    return Math.floor(Math.random() * Math.floor(max));
+  }
+
+  function createTemporaryRuleset(): Promise<admin.securityRules.Ruleset> {
+    const name = 'random_name_' + getRandomInt(100000) + '.rules';
```

Comment:
Rules file names don't really have to be unique across rulesets. Feel free to use `firestore.rules`.

Reply:
This wasn't to make them unique across rulesets, but rather for the following:
* it allows you to run tests in parallel without them tripping over each other. (Theoretical benefit only; I don't think mocha allows parallel execution of tests.)
* it allows you to call it twice to generate two different rulesets, like I'm doing on line 258, now 269... unless you get really unlucky, in which case the test will flake. Yikes. Fixed.
* it ensures`[*]` that a test that accidentally leaves a rules file behind doesn't impact the other tests. (`[*]` - or makes it unlikely to occur.)

I've added a comment to this effect, referencing the last bullet point.